contrib/logging/fluentd-sidecar-gcp/README.md

```diff
@@ -0,0 +1,22 @@
+# Collecting log files from within containers with Fluentd and sending to the Google Cloud Logging service.
+This directory contains the source files needed to make a Docker image that collects log files from arbitrary files within a container using [Fluentd](http://www.fluentd.org/) and sends them to GCP.
+The image is designed to be used as a sidecar container as part of a pod.
+It lives in the Google Container Registry under the name `gcr.io/google_containers/fluentd-sidecar-gcp`.
+
+This shouldn't be necessary if your container writes its logs to stdout or stderr, since the Kubernetes cluster's default logging infrastructure will collect that automatically, but this is useful if your application logs to a specific file in its filesystem and can't easily be changed.
+
+In order to make this work, you have to add a few things to your pod config:
+1. A second container, using the `gcr.io/google_containers/fluentd-sidecar-gcp:1.0` image to send the logs to Google Cloud Logging.
```

Comment:
Please can this be formatted as a list with each item starting on a new line? Thanks.